README.md

```diff
@@ -1,12 +1,23 @@
 <p align="center">
-<a href="https://dagster.io/"><img src="assets/dagster-logo.png"></a>
-<br /><br />
-<a href="https://twitter.com/dagsterio"><img src="https://img.shields.io/twitter/follow/dagsterio"></a>
-<a href="https://dagster.io/slack"><img src="https://dagster-slackin.herokuapp.com/badge.svg"></a>
-<a href="https://github.com/dagster-io/dagster"><img src="https://img.shields.io/github/stars/dagster-io/dagster?label=Star&style=social"></a>
+  <a target="_blank" href="https://dagster.io">
+    <picture>
+        <source media="(prefers-color-scheme: dark)" srcset=".github/dagster-logo-dark.svg">
+        <img alt="dagster logo" src=".github/dagster-logo-light.svg" width="auto" height="90">
```

Comment:
We should switch to absolute urls before merging, so that this renders properly on other mirrors.
```suggestion
        <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/dagster-io/dagster/master/.github/dagster-logo-dark.svg">
        <img alt="dagster logo" src="https://raw.githubusercontent.com/dagster-io/dagster/master/.github/dagster-logo-light.svg" width="auto" height="90">
```